SingularityUI/app/views/tail.coffee

```diff
@@ -176,7 +178,8 @@ class TailView extends View
         @$el.addClass 'tailing'
 
     stopTailing: ->
-        return if @isTailing isnt true
+        task = @model.get('taskUpdates')[..].pop()
```

Comment:
sorry, don't know what i was thinking there. could've just been `arr[arr.length-1]` but cool that there's a _.last() helper